alicloud resource Provisioning fails randomly

**What happened:**

Provisioning of alicloud resources fails randomly with an already existing bucket.

**Debug Logs:**

https://github.com/gardenlinux/gardenlinux/actions/runs/17828465582/job/50688552708

```
 │ Warning: "name": [DEPRECATED] Field `name` has been deprecated from provider version 1.239.0. New field `security_group_name` instead.
│ 
│   with module.ali.alicloud_security_group.sg,
│   on modules/ali/main.tf line 143, in resource "alicloud_security_group" "sg":
│  143: resource "alicloud_security_group" "sg" {
│ 
│ (and 2 more similar warnings elsewhere)
╵
╷
│ Warning: Deprecated attribute
│ 
│   on .terraform/modules/state_aws.terraform_state_backend/main.tf line 22, in locals:
│   22:     region = data.aws_region.current.name
│ 
│ The attribute "name" is deprecated. Refer to the provider documentation for details.
╵
╷
│ Error: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_oss_bucket_public_access_block.go:78: Resource alicloud_oss_bucket_public_access_block /?publicAccessBlock Failed!!! [SDK alibaba-cloud-sdk-go ERROR]:
│ SDKError:
│    StatusCode: 409
│    Code: BucketAlreadyExists
│    Message: The requested bucket name is not available. The bucket namespace is shared by all users of the system. Please select a different name and try again.
│    Data: {"AccessDeniedDetail":null,"Recommend":"https://api.alibabacloud.com/troubleshoot?q=0015-00000102","ecCode":"0015-00000102","hostId":"images-gh-actions-ali-gardener-prod-amd64-a242708d.oss-eu-west-1.aliyuncs.com","requestId":"68CBFD9E2D4DE739373EACFA","statusCode":409}
│ 
│ 
│   with module.ali["ali-gardener_prod-amd64"].alicloud_oss_bucket_public_access_block.no_public_access[0],
│   on modules/ali/main.tf line 95, in resource "alicloud_oss_bucket_public_access_block" "no_public_access":
│   95: resource "alicloud_oss_bucket_public_access_block" "no_public_access" {
│ 
╵
```

--- a/tests/platformSetup/tofu/modules/ali/main.tf
+++ b/tests/platformSetup/tofu/modules/ali/main.tf
@@ -141,7 +141,7 @@ resource "alicloud_vswitch" "subnet" {
 }
 
 resource "alicloud_security_group" "sg" {
-  name   = local.fw_name
+  security_group_name = local.fw_name
   vpc_id = alicloud_vpc.net.id
 
   tags = merge(

